h2o-algos/src/test/java/hex/pdp/PartialDependenceTest.java

```diff
@@ -121,6 +118,127 @@ assert equalTwoDimTables(partialDependence._partial_dependence_data[1], partialD
     }
   }
 
+  /**
+   * This test will test the 2d pdp implementation for 2 sets of column pairs.  In order to reduce the test runtime, 
+   * I will choose to specify a few split-points.
+   */
+  @Test public void prostate2Dpdp() {
+    Scope.enter();
+    double tot = 1e-10;
+    try {
+      // Frame
+      Frame fr = parse_test_file("smalldata/prostate/prostate.csv");
+      for (String s : new String[]{"RACE","GLEASON","DPROS","DCAPS","CAPSULE"}) { // convert to enum columns
+        Vec v = fr.remove(s);
+        fr.add(s, v.toCategoricalVec());
+        v.remove();
+      }
+      Scope.track(fr);
+      
+      // Build model Model
+      GBMModel.GBMParameters parms = new GBMModel.GBMParameters();
+      parms._train = fr._key;
+      parms._ignored_columns = new String[]{"ID"};
+      parms._response_column = "CAPSULE";
+      GBMModel model = new GBM(parms).trainModel().get();
+      
+      // PartialDependence with only 1D pdp
+      PartialDependence partialDependence1 = new PartialDependence(Key.<PartialDependence>make());
+      partialDependence1._nbins = 10;
+      partialDependence1._model_id = (Key) model._key;
+      partialDependence1._cols = new String[]{"RACE", "VOL"};
+      partialDependence1._frame_id = fr._key;
+      partialDependence1.execImpl().get();
+      
+      // PartialDependence with 1D and 2D pdp
+      PartialDependence partialDependence = new PartialDependence(Key.<PartialDependence>make());
+      partialDependence._nbins = 10;
+      partialDependence._model_id = (Key) model._key;
+      partialDependence._cols = new String[]{"RACE", "VOL"};
+      partialDependence._col_pairs_2dpdp = new String[2][];
+      partialDependence._col_pairs_2dpdp[0] = new String[]{"AGE", "RACE"};
+      partialDependence._col_pairs_2dpdp[1] = new String[]{"AGE", "PSA"};
+      partialDependence._user_cols = new String[]{"AGE", "PSA"};
+      partialDependence._num_user_splits = new int[] {3,3};
+      partialDependence._user_splits_present = true;
+      partialDependence._user_splits = new double[]{65, 61, 72, 1.4, 6.7, 20};
+      partialDependence._frame_id = fr._key;
+      partialDependence.execImpl().get();
+      
+      Scope.track_generic(model);
+      Scope.track_generic(partialDependence);
+      Scope.track_generic(partialDependence1);
+      
+      // Compare 1d pdp results with and without 2D pdp.  They should be the same.
+      assert equalTwoDimTables(partialDependence._partial_dependence_data[0], partialDependence1._partial_dependence_data[0], 1e-10):
```

Comment:
There is `double tot = 1e-10;` defined at the very beginning (multiple occurances).

Reply:
probably should define it at the top for everyone then.  I will put it in.  Thanks.